poetry/layouts/layout.py

```diff
@@ -83,17 +94,21 @@ def create(self, path, with_tests=True):
 
     def generate_poetry_content(self):
         template = POETRY_DEFAULT
-        if self._license:
-            template = POETRY_WITH_LICENSE
 
         content = loads(template)
+
         poetry_content = content["tool"]["poetry"]
         poetry_content["name"] = self._project
         poetry_content["version"] = self._version
         poetry_content["description"] = self._description
         poetry_content["authors"].append(self._author)
+
         if self._license:
             poetry_content["license"] = self._license
+        else:
+            poetry_content.remove("license")
+
+        poetry_content["packages"].append(self.get_package_include())
```

Comment:
Hm, according to your PR info, it is intended to include `packages` in any case. I like the current behavior more, to only use it, if the structure or name differs from the default. The reason for this is, that it looks to me like a redundancy to say "I have a package called x, please include x". But as I said: Just a personal opinion :)

Reply:
I thought about that, however I believe that being explicit is a better practice here. Quite often we endup with bad usages because the development path included auto-discovered packages. There are cases where the name and package name are different (`a-b-c` vs `a.b.c` vs `a_b_c`). Additionally, it allows for easier editing if required as opposed to having to look up the option etc. 

We can discard it if the project name and package name are the same maybe?

Reply:
> We can discard it if the project name and package name are the same maybe?

That would be something, I can live with :+1: